SpatialGDK/Documentation/content/singleton-actors.md

```diff
@@ -1,55 +1,84 @@
 <%(TOC)%>
 # Singleton Actors
 
-Singleton Actors allow a single source of truth for both operations and data across a multiserver simulation. They are server-side authoritative [Unreal Actors](https://docs.unrealengine.com/en-us/Programming/UnrealArchitecture/Actors) that are restricted to one instantiation on SpatialOS. For example, if you are implementing a scoreboard, you'd most likely only want there to be one of them in your world. Ensuring this behavior in a multiserver paradigm requires a few additional steps which can be easily facilitated through the Singleton Actor.
+You can use a Singleton Actor to define a single source of truth for operations or data, or both, across a game world that uses zoning. You can have only one instance of an [entity]({{urlRoot}}/content/glossary#entity) that represents a Singleton Actor per game world.
 
-There are two kinds of Singleton Actors:
+You define a Singleton Actor by tagging an Actor with the `SpatialType=Singleton` class attribute. For example, if you are implementing a scoreboard, you probably want only one scoreboard in your world, so you can tag the scoreboard Actor as a Singleton Actor.
 
-* **Public Singleton Actors** - Singleton Actors which are replicated to [server-workers and client-workers]({{urlRoot}}/content/glossary#workers). [AGameState](https://docs.unrealengine.com/en-US/Gameplay/Framework/GameMode) is a Public Singleton Actor.
-* **Private Singleton Actors** - Singleton Actors which are replicated to [server-workers]({{urlRoot}}/content/glossary#workers), but not accessible to [client-workers]({{urlRoot}}/content/glossary#workers). [AGameMode](https://docs.unrealengine.com/en-US/Gameplay/Framework/GameMode) is a Private Singleton Actor.
+There are two Singleton Actor types:
```

Comment:
I think this is OK here. However, I don't particularly like it, because it makes me think that there might be a thing called a "Singleton Actor type". But it's not a big deal - I think people will understand what it means :)